sensor/kubernetes/certrefresh/cert_refresher.go

```diff
@@ -96,21 +95,21 @@ func getServiceTypeNames(serviceCertificates []*storage.TypedServiceCertificate)
 	return serviceTypeNames
 }
 
-func getTimeToRefreshFromRepo(ctx context.Context, getCertsRenewalTime getCertsRenewalTimeFunc,
+func getTimeToRefreshFromRepo(ctx context.Context, certsDescription string, getCertsRenewalTime getCertsRenewalTimeFunc,
 	repository certrepo.ServiceCertificatesRepo) (time.Duration, error) {
 
 	certificates, getCertsErr := repository.GetServiceCertificates(ctx)
 	if errors.Is(getCertsErr, certrepo.ErrUnexpectedSecretsOwner) {
 		return 0, getCertsErr
 	}
 	if errors.Is(getCertsErr, certrepo.ErrDifferentCAForDifferentServiceTypes) || errors.Is(getCertsErr, certrepo.ErrMissingSecretData) {
-		log.Errorf("local scanner certificates are in an inconsistent state, "+
-			"will refresh certificates immediately: %s", getCertsErr)
+		log.Errorf("%s are in an inconsistent state, "+
```

Comment:
Looking at how this `certsDescription` is used I am wondering if we could maybe add this information somehow directly to the logger so that we don't need to inject it into the actual log messages every time.

Reply:
I'm not sure how this would work, my (very rough) understandings of the logger is that there's one logger object per module (i.e. package?) but in this case `certsDescription` is not always the same. 

You can call `newCertificatesRefresher` with different `certsDescription`, now there's only one but there will be another soon.